source/includes/sample-data-note.rst

```diff
@@ -0,0 +1,5 @@
+.. note::
```

Comment:
Should these be in the PR if you're not using them anymore?

Reply:
They should be removed. I wanted to get your feedback on the note before deleting it as I think the note in the Go Driver FindOne.txt page should be replaced with what we worked on here. I will delete both notes before merging. Fix: https://github.com/mongodb/docs-node/pull/167/commits/1b8fafc56e1fa802114d687177bdddfa26016789